src/oatpp-swagger/Generator.cpp

```diff
@@ -421,7 +421,7 @@ void Generator::generatePathItemData(const std::shared_ptr<Endpoint>& endpoint,
         for (const auto &header : info->headers.getOrder()) {
           // We don't want the Authorization header listed as Parameter. This should be done in ENDPOINT_INFO() { info->addSecurityRequirement( /* SecurityScheme-Name */ ); }
           if (header != oatpp::web::protocol::http::Header::AUTHORIZATION) {
-            filteredHeaders[header] = info->headers[header];
+            filteredHeaders.add(header, info->headers[header].type) = info->headers[header];
```

Comment:
Hello @asiirtol ,

I believe that it's better to fix it here:
- https://github.com/oatpp/oatpp/blob/master/src/oatpp/web/server/api/Endpoint.cpp#L52
- and here https://github.com/oatpp/oatpp/blob/master/src/oatpp/web/server/api/Endpoint.cpp#L44

The `[]` operator should check - if it's a new element - then add to `m_order`

Reply:
I agree that modifying operator[] would be a more elegant solution but operator[] only takes a name as a parameter. In order to add a new element it should know the type of the element, too. How operator[] could do that?